mittab/libs/data_import/export_xls_files.py

```diff
@@ -0,0 +1,220 @@
+from django.contrib.auth.decorators import permission_required
+from django.http import HttpResponse
+from xlwt import Workbook
+
+from mittab.apps.tab.models import Team, Judge, Room, Debater
+from mittab.libs import tab_logic
+
+
+def _create_vn_str(novice_status):
+    """Creates varsity-novice status string from the integer pseudo-enum used by the model"""
+    if novice_status is 0: return "varsity"
+    if novice_status is 1: return "novice"
+    return novice_status
+
+
+@permission_required('tab.tab_settings.can_change', login_url="/403/")
+def export_teams(request):
```

Comment:
These functions shouldn't be taking requests and returning responses. Have the `/data_import` files just return the excel object, and write another view function to handle this. Otherwise it's mixing web app logic into a library that should just be doing excel stuff

Reply:
Same goes for the other functions in this package